src/resolvers/versionedTemplate.ts

```diff
@@ -22,7 +22,7 @@ export const resolvers: Resolvers = {
     // Search for PublishedTemplates whose name or owning Org's name contains the search term
     //    - called by the Template Builder - prior template selection page
     publishedTemplates: async (_, { term }, context: MyContext): Promise<VersionedTemplate[]> => {
-      if (isAdmin(context.token)){
+      if (isAdmin(context.token)) {
         return await VersionedTemplate.search('publishedTemplates resolver', context, term);
       }
       // Unauthorized!
```

Comment:
Should add chaining here too go the versionedTemplate can get at all of the versionedSections

Reply:
That makes sense. Will update the schema for versionedTemplate to include versionedSection and add the resolver chaining.

Reply:
Hi @briri I added chaining for versionedSection to versionedTemplate resolver, and also moved the logic for determining what new tags to add to sectionTags into sectionService.